Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>rootProject.name = 'minecraft-launcher'\r\ninclude 'plugin'\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision 39aa137ccafa784e52abb0686a20de6900268df5)
+++ b/settings.gradle	(date 1627373144834)
@@ -1,3 +1,3 @@
 rootProject.name = 'minecraft-launcher'
 include 'plugin'
-
+include 'app'
\ No newline at end of file
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Project exclude paths\r\n/.gradle/\r\n/build/\r\n/build/classes/java/main/\r\n/clients/\r\n/players/\r\n/received_clients_tmp/\r\n/loaded_clients/\r\n/out/\r\n/logs/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 39aa137ccafa784e52abb0686a20de6900268df5)
+++ b/.gitignore	(date 1627376249870)
@@ -1,10 +1,3 @@
 # Project exclude paths
 /.gradle/
 /build/
-/build/classes/java/main/
-/clients/
-/players/
-/received_clients_tmp/
-/loaded_clients/
-/out/
-/logs/
\ No newline at end of file
Index: .idea/modules/app/minecraft-launcher.app.test.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules/app/minecraft-launcher.app.test.iml b/.idea/modules/app/minecraft-launcher.app.test.iml
new file mode 100644
--- /dev/null	(date 1627373132428)
+++ b/.idea/modules/app/minecraft-launcher.app.test.iml	(date 1627373132428)
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="TestModuleProperties" production-module="minecraft-launcher.app.main" />
+</module>
\ No newline at end of file
Index: app/src/main/java/com/Test.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/Test.kt b/app/src/main/java/com/Test.kt
new file mode 100644
--- /dev/null	(date 1627373246395)
+++ b/app/src/main/java/com/Test.kt	(date 1627373246395)
@@ -0,0 +1,9 @@
+package com
+
+fun main(){
+    println("Test")
+}
+
+class Test {
+
+}
\ No newline at end of file
Index: .idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"GradleMigrationSettings\" migrationVersion=\"1\" />\r\n  <component name=\"GradleSettings\">\r\n    <option name=\"linkedExternalProjectsSettings\">\r\n      <GradleProjectSettings>\r\n        <option name=\"delegatedBuild\" value=\"true\" />\r\n        <option name=\"testRunner\" value=\"GRADLE\" />\r\n        <option name=\"distributionType\" value=\"DEFAULT_WRAPPED\" />\r\n        <option name=\"externalProjectPath\" value=\"$PROJECT_DIR$\" />\r\n        <option name=\"modules\">\r\n          <set>\r\n            <option value=\"$PROJECT_DIR$\" />\r\n            <option value=\"$PROJECT_DIR$/plugin\" />\r\n          </set>\r\n        </option>\r\n      </GradleProjectSettings>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/gradle.xml b/.idea/gradle.xml
--- a/.idea/gradle.xml	(revision 39aa137ccafa784e52abb0686a20de6900268df5)
+++ b/.idea/gradle.xml	(date 1627373154660)
@@ -11,6 +11,7 @@
         <option name="modules">
           <set>
             <option value="$PROJECT_DIR$" />
+            <option value="$PROJECT_DIR$/app" />
             <option value="$PROJECT_DIR$/plugin" />
           </set>
         </option>
Index: app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/build.gradle b/app/build.gradle
new file mode 100644
--- /dev/null	(date 1627375802442)
+++ b/app/build.gradle	(date 1627375802442)
@@ -0,0 +1,17 @@
+plugins {
+    id 'java'
+    id 'org.jetbrains.kotlin.jvm'
+}
+apply plugin: "ru.mail.husker-dev.native"
+
+group 'com.husker.minecraft.launcher'
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+
+}
+
